pages/docs/reason-compiler/latest/decorators.mdx

```diff
@@ -525,7 +525,16 @@ var myBook = new Book();
 
 ### `[@bs.deriving accessors]`
 
-Reason records are represented as arrays in JS.
+Reason records were represented as arrays in JS in BuckleScript versions < v7. `bs.deriving accessors` makes it more convenient to access records when you or your library consumers need to be on an older BuckleScript version.
+
+<Info>
+
+As records are compiled to JS objects since BuckleScript v7, you may not need this decorator anymore, except if you need to stay compatible for users with older BuckleScript versions (such as library authors).
```

Comment:
Maybe only put a hint in there to point them to one place where interop is being discussed?

```suggestion
**Hint:** Since BuckleScript v7, records are compiled to plain JS objects, so you might not need [@bs.deriving accessors] / [@bs.deriving abstract]. More about this [here](/docs/reason-compiler/latest/object#records-as-objects).
```

Reply:
This could eventually be part of the `[bs.deriving]` subheader

Reply:
Agree, I would add on the accessor that publishes each key as a function to get the value and another of the setter (and this mutates the object).

https://github.com/moroshko/bs-blabla/blob/master/bs.deriving.md#bsderiving